galaxy_api/contrib/logging.py

```diff
@@ -0,0 +1,26 @@
+"""Logging utils."""
```

Comment:
Module `logging` will not conflict, because we use absolute imports and python supports aliases.
Ideally it should be `contrib.logging.cloud_watch`, but I didn't want to create too nested structure unless we need so. It can be refactored later if needed.